specification/eventgrid/data-plane/Microsoft.ResourceNotifications/stable/2018-01-01/HealthResources.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
What happened to the other swagger files?

Reply:
we are currently declaring public preview only for health resources system topic. So excluding the other files for now.